cpu/sam0_common/periph/pwm.c

```diff
@@ -45,33 +87,34 @@ static uint8_t _get_prescaler(unsigned int target, int *scale)
 
     if (target >= 512) {
         *scale = 1024;
-        return TCC_CTRLA_PRESCALER_DIV1024_Val;
+        return TC_CTRLA_PRESCALER_DIV1024_Val;
```

Comment:
are we sure this change doesn't affect TCC operation for others SAM0 ?

Reply:
I could add a 

```C
#ifdef TCC_CTRLA_PRESCALER_DIV1024_Val
    static_assert(TCC_CTRLA_PRESCALER_DIV1024_Val == TC_CTRLA_PRESCALER_DIV1024_Val);
#endif
```

For all families supported right now both defines are equal (checked by a quick [`rg`](https://github.com/BurntSushi/ripgrep))

Reply:
There is already too much #ifdef but I guess we don't have the choice here...